src/tests/flyer.test.ts

```diff
@@ -276,6 +276,45 @@ describe('getFlyersByCategorySlug tests', () => {
   });
 });
 
+describe('getAllFlyerCategories tests', () => {
```

Comment:
I would put all of this at the end of the file just to keep things in order of implementation but not a big deal.